src-tauri/src/main.rs

```diff
@@ -34,18 +36,23 @@ use mod_management::{
     get_installed_mods_and_properties, set_mod_enabled_status,
 };
 
+mod plugin_management;
+use crate::plugin_management::{receive_install_status, InstallStatusSender};
+
 mod northstar;
 use northstar::get_northstar_version_number;
 
 mod thunderstore;
 use thunderstore::query_thunderstore_packages_api;
 
-use tauri::{Manager, Runtime};
+use tauri::{AppHandle, Manager, Runtime};
 use tokio::time::sleep;
 
 #[derive(Default)]
 struct Counter(Arc<Mutex<i32>>);
 
+pub static APP_HANDLE: OnceCell<AppHandle> = OnceCell::new();
```

Comment:
My Rust knowledge is a bit limited. This essentially sets a global static var, right?

I really could've used this when implementing showing download progress updates in front-end, haha ^^"

Reply:
yes, it gets initialized here btw https://github.com/R2NorthstarTools/FlightCore/blob/5fb21077e40daca339ae917d7efa9cd1c19a08ec/src-tauri/src/main.rs#L118